rush-plugins/rush-litewatch-plugin/config/rush-project.json

```diff
@@ -0,0 +1,8 @@
+{
+  "operationSettings": [
```

Comment:
Why not just use `heft-node-rig`?

Reply:
Ah you are. This file shouldn't be necessary.

Reply:
This is a copy+paste of the same `rush-project.json` used by all the other projects in this repo.  Should we delete it for all projects?  (Maybe in a separate PR?)

Reply:
Maybe those projects have cyclic dependencies.  I removed this file and there weren't any build cache warnings.